#
# Service to get the correction angle
#

---
float32 correction_angle